packages/client/hmi-client/src/components/widgets/tera-input-text.vue

```diff
@@ -11,20 +11,22 @@
 				type="text"
 				:value="displayValue()"
 				@click.stop
-				@focus="isFocused = true"
+				@focus="onFocus"
 				@focusout="$emit('focusout', $event)"
 				@blur="onBlur"
 				@input="updateValue"
+				@keyup="$emit('keyup', $event)"
+				@keypress="$emit('keypress', $event)"
```

Comment:
Maybe we should forward-port to use the change event in tera-input, rather than back-port tera-input to support these events?

Reply:
As I replace the primevue text components - Im just trying not too disturb what we currently have.  There are some primevue components which listen to these events, Im just sending them up (keyup.esc, keyup.enter).  Perhaps once I've moved over all the components it might be worth it to re-evaluate whether we should have these events?